docs/tutorials/notebooks/basic_ot_between_datasets.ipynb

```diff
@@ -0,0 +1,398 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    plt.quiver(y[:, 0], y[:, 1], -gradients[:, 0], -gradients[:, 0])
</pre><p>,1 for second, here gradients are on the y=x axis</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/296/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>